docs/guides/weaviate/README.md

```
@@ -0,0 +1,54 @@
+    identifier: weaviate-readme
+    name: Weaviate
+    parent: weaviate-guides
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+url: /docs/{{ .version }}/guides/weaviate/
+aliases:
+  - /docs/{{ .version }}/guides/weaviate/README/
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)